src/components/Welcome/index.tsx

```diff
@@ -68,24 +70,41 @@ const WelcomeActions: FunctionComponent<WelcomeActionsProps> = ({
         onClick={nextStep}
         data-onfido-qa="welcome-next-btn"
       >
-        {welcomeNextButton}
+        {buttonLabel}
       </Button>
     </div>
   )
 }
 
-const Welcome: FunctionComponent<StepComponentBaseProps> = ({ nextStep }) => {
+const Welcome: FunctionComponent<StepComponentBaseProps> = ({
+  steps,
+  nextStep,
+}) => {
   const [, { findStep }] = useSdkOptions()
   const { translate } = useLocales()
-  const { title, descriptions, nextButton } = findStep('welcome')?.options || {}
+  const {
+    title: customTitle,
+    descriptions: customDescriptions,
+    nextButton: customNextButtonLabel,
+  } = findStep('welcome')?.options || {}
+
+  const actions = <WelcomeActions {...{ customNextButtonLabel, nextStep }} />
+  const welcomeTitle = customTitle ? customTitle : translate('welcome.title')
+  const welcomeSubTitle = !customDescriptions
+    ? translate('welcome.subtitle')
+    : ''
 
-  const actions = <WelcomeActions {...{ nextButton, nextStep }} />
-  const welcomeTitle = title ? title : translate('welcome.title')
+  const configuredIDVSteps = steps
+    .filter((step) => step.type === 'document' || step.type === 'face')
+    .map((stepConfig) => stepConfig.type)
+  const welcomeDescriptions = customDescriptions
+    ? customDescriptions
+    : getLocalisedDescriptions(configuredIDVSteps, translate)
```

Comment:
What will this display for `poa` or `auth`? Shouldn't we have a default for those cases?

Reply:
Good point. I'm not sure about messaging for `auth` or how to preview it so will need to catch up with @carlagn but for `poa` we do need some kind of default messaging in English.

Reply:
I've mapped the `auth` Welcome screen copy to `face` for now. Auth is currently just using the existing SDK's welcome screen but will bring it up on Slack for discussion with designers.